### PR TITLE
Node expansion: Skipping Vmware code

### PR DESCRIPTION
**Pull request by shreekarSS**

### CODE REVIEW

File: tests/manage/z_cluster/cluster_expansion/test_node_expansion.py [hunk 1]
```diff
@@ -15,25 +16,34 @@ class TestAddNode(ManageTest):
...
         """
-        dt = config.ENV_DATA['deployment_type']
-        if dt == 'ipi':
-            machines = machine_utils.get_machinesets()
-            logger.info(f'The worker nodes number before expansion {len(helpers.get_worker_nodes())}')
-            for machine in machines:
-                add_new_node_and_label_it(machine)
-            logger.info(f'The worker nodes number after expansion {len(helpers.get_worker_nodes())}')
+        new_nodes = 3
+        if config.ENV_DATA['platform'] == 'AWS':
```
Comment (vavuthu) [inline tests/manage/z_cluster/cluster_expansion/test_node_expansion.py:24]: use constants.AWS_PLATFORM

File: tests/manage/z_cluster/cluster_expansion/test_node_expansion.py [hunk 2]
```diff
@@ -15,25 +16,34 @@ class TestAddNode(ManageTest):
...
+                logger.info(f'The worker nodes number after expansion {len(helpers.get_worker_nodes())}')
 
-        else:
-            new_nodes = 3
+        elif config.ENV_DATA['platform'] == 'vsphere':
             logger.info(f'The worker nodes number before expansion {len(helpers.get_worker_nodes())}')
-            if config.ENV_DATA.get('rhel_workers'):
-                node_type = 'RHEL'
+            if config.ENV_DATA.get('rhel_user'):
+                node_type = constants.RHEL_OS
```
Comment (vavuthu) [inline tests/manage/z_cluster/cluster_expansion/test_node_expansion.py:45]: Do we have implementation for RHEL nodes in vSphere platform. If not we need skip this for now

File: tests/manage/z_cluster/cluster_expansion/test_node_expansion.py (hunk 1 above)
Comment (shreekarSS) [reply tests/manage/z_cluster/cluster_expansion/test_node_expansion.py:24]: ack

File: tests/manage/z_cluster/cluster_expansion/test_node_expansion.py (hunk 2 above)
Comment (shreekarSS) [reply tests/manage/z_cluster/cluster_expansion/test_node_expansion.py:45]: yes, will skip .

**Comment by shreekarSS** [comment]
Pr testing is blocked with bug https://bugzilla.redhat.com/show_bug.cgi?id=1844521

**Comment by shreekarSS** [comment]
Verified on aws 1az

07:43:59 - MainThread - tests.manage.z_cluster.cluster_expansion.test_node_expansion - INFO - The worker nodes number after expansion 4
PASSED                                                 [100%]
--------------------- live log teardown ----------------------
07:43:59 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc wait --for condition=ready pod -l app=rook-ceph-tools -n openshift-storage --timeout=120s
07:43:59 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage get pod -l 'app=rook-ceph-tools' -o jsonpath='{.items[0].metadata.name}'
07:43:59 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage exec rook-ceph-tools-b7789c95-qrc9g ceph health
07:44:00 - MainThread - ocs_ci.utility.utils - WARNING - Command stderr: kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl kubectl exec [POD] -- [COMMAND] instead.

07:44:00 - MainThread - ocs_ci.utility.utils - INFO - Ceph cluster health is HEALTH_OK.
07:44:00 - MainThread - tests.conftest - INFO - Ceph health check passed at teardown


====================== warnings summary ======================
/home/jenkins/venv/lib/python3.6/site-packages/_pytest/config/__init__.py:891
  /home/jenkins/venv/lib/python3.6/site-packages/_pytest/config/__init__.py:891: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: pytest_reportportal
    self._mark_plugins_for_rewrite(hook)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
========== 1 passed, 1 warning in 310.19s (0:05:10) ==========

File: tests/manage/z_cluster/cluster_expansion/test_node_expansion.py [hunk 3]
```diff
@@ -16,25 +17,34 @@ class TestAddNode(ManageTest):
...
+                logger.info(f'The worker nodes number before expansion {len(helpers.get_worker_nodes())}')
+                if config.ENV_DATA.get('rhel_workers'):
+                    node_type = constants.RHEL_OS
+                else:
+                    node_type = constants.RHCOS
+                assert add_new_node_and_label_upi(node_type, new_nodes), "Add node failed"
+                logger.info(f'The worker nodes number after expansion {len(helpers.get_worker_nodes())}')
+
+        elif config.ENV_DATA['platform'].lower() == constants.VSPHERE_PLATFORM:
+            pytest.skip(" Skipping add node in Vmware pf due to https://bugzilla.redhat.com/show_bug.cgi?id=1844521")
```
Comment (vavuthu) [inline tests/manage/z_cluster/cluster_expansion/test_node_expansion.py:44]: removing space in the begining of line and replace pf with platform

Comment (shreekarSS) [reply tests/manage/z_cluster/cluster_expansion/test_node_expansion.py:44]: ack

**Comment by shreekarSS** [comment]
> Pr testing is blocked with bug https://bugzilla.redhat.com/show_bug.cgi?id=1844521

Skipping add node for vmware platform

**Pull request merged**